guidelines/sc/22/target-size-minimum.html

```diff
@@ -11,7 +11,7 @@ <h4>Target Size (Minimum)</h4>
         <li><strong>Inline:</strong> The target is in a sentence or block of text;</li>
         <li><strong>Necessary:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</li>
         <li><strong>Legal:</strong> A particular presentation of the target is legally required.</li>
-        <li><strong>Alternative:</strong> The function can be acheived through a different control on the same page that has an area of at least 24 by 24 CSS pixels;</li>
+        <li><strong>Equivalent:</strong> The function can be acheived through a different control on the same page that has an area of at least 24 by 24 CSS pixels;</li>
```

Comment:
although this has already been merged, note the pre-existing typo `acheived` > `achieved`

Reply:
Since this is merged, I corrected it in the unmerged PR of the same file. https://github.com/w3c/wcag/pull/2330/files

Reply:
👍